spec/features/votes/user_votes_on_review_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+feature 'creating and updating vote' do
+  context 'as an authorized user' do
+    let(:user) { FactoryGirl.create(:user) }
+    let(:restaurant) { FactoryGirl.build(:restaurant, user: user) }
+
+    before :each do
+      sign_in_as user
+    end
+
+    scenario 'I can successfully up vote a review' do
+      review = FactoryGirl.create(:review, restaurant: restaurant)
+      visit review_path(review)
+
+      click_button 'Upvote Review'
+
+      expect(page).to have_content('Upvote saved!')
+      expect(page).to have_content(restaurant.name)
+      expect(page).to have_content(review.body)
+    end
+
+    scenario 'I can successfully down vote a review' do
+      review = FactoryGirl.create(:review, restaurant: restaurant)
+      visit review_path(review)
+
+      click_button 'Downvote Review'
+
+      expect(page).to have_content('Downvote saved!')
+      expect(page).to have_content(restaurant.name)
+      expect(page).to have_content(review.body)
+    end
+
+    scenario 'I cannot up vote a review multiple times' do
+      review = FactoryGirl.create(:review, restaurant: restaurant)
+      visit review_path(review)
+
+      click_button 'Upvote Review'
+      click_button 'Upvote Review'
```

Comment:
Wouldn't it make more sense to create a vote related factory that gets generated and then trying clicking 'Upvote Review'? Right now, this test depends on your voting functionality working and that isn't a good idea. Each test should test one particular atomic feature.